### ISSUE TITLE
email to marinemammals when data request made

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-tools-dev/cetaceans#274]
not been received, Carlos to look at marinemammals@jncc.gov.uk

**Comment by NikiClear** [comment]
Tested, emailed came to niki.clear@ as the requester but nothing has come through to marinemamamls@ 

In my request message, I have asked to data custodian to confirm they have received the notification to check whether this step is working.

**Comment by NikiClear** [comment]
Another issue: 
When a request for data has been approved, the email notification says

"Thank you for your request to access the restricted Cetaceans data records. Your request has been updated.

You can check the status per survey at: [https://cetaceans.ices.dk/Download/d594e0d4-de6c-456c-a65f-f7cc07c90370](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcetaceans.ices.dk%2FDownload%2Fd594e0d4-de6c-456c-a65f-f7cc07c90370&data=05%7C01%7CNiki.Clear%40jncc.gov.uk%7C306daa264b7d4be0996008da8f18af67%7C444ee4e8b2fd491d8c318b0508370a6b%7C0%7C0%7C637979633428917437%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=y2B%2FwrrPw748f%2ByOeg3DMZ3IaU4P45RLbTZJgfOi1qk%3D&reserved=0)"

However, when I follow the link, it is not obvious on the page what the change has been and there is no option to download data. 
https://cetaceans.ices.dk/Download/d594e0d4-de6c-456c-a65f-f7cc07c90370

**Comment by cmspinto** [comment]
there is no link to download because the data hasn't been released yet. 

Should I try to approve the request?

**Comment by NikiClear** [comment]
> there is no link to download because the data hasn't been released yet.
> 
> Should I try to approve the request?

OK. But why did the system create an email stating "Your request has been updated." Was the request rejected? If the request was rejected, should there be something on the page to stat that?

**Comment by cmspinto** [comment]
Hi Niki, sorry. 

Just checked and the request was approved :).
So the email is right, the bug is somewhere else, will check that now.

![image](https://user-images.githubusercontent.com/8705477/188585545-10a02b60-5f31-474a-9413-3abcbd9f0860.png)

**Comment by cmspinto** [comment]
Discover the bug. The user was not a manager of ORCA organization. 

Working now: https://cetaceans.ices.dk/Download/d594e0d4-de6c-456c-a65f-f7cc07c90370

Still need to find out how the user was able to approve the request!

**Comment by cmspinto** [comment]
An email should have been sent now with a test request. @NikiClear did you get the test email?

**Comment by NikiClear** [comment]
> An email should have been sent now with a test request. @NikiClear did you get the test email?

yep, that has come through to marinemammals@. Thanks for checking, think this one can be closed now :)
